geoportal/c2cgeoportal_geoportal/views/entry.py

```diff
@@ -255,7 +270,9 @@ def _wms_getcap_cached(self, ogc_server, _):
             headers.pop("Host")
 
         try:
-            response = self.get_http_cached(url, headers)
+            response = await asyncio.get_event_loop().run_in_executor(
+                None, get_http_cached, self.http_options, url, headers
+            )
```

Comment:
What does this do exactly, is it not the same as : 
```
response = await get_http_cached(self.http_options, url, headers)
```

Reply:
No, we need a separate thread for get_http_cached...